templates/15_insert.tpl

```diff
@@ -66,7 +70,15 @@ func (o *{{$tableNameSingular}}) Insert(exec boil.Executor, whitelist ... string
 			return err
 		}
 		if len(wl) != 0 {
-			cache.query = fmt.Sprintf("INSERT INTO {{$schemaTable}} ({{.LQ}}%s{{.RQ}}) VALUES (%s)", strings.Join(wl, "{{.LQ}},{{.RQ}}"), strmangle.Placeholders(dialect.IndexPlaceholders, len(wl), 1, 1))
+			{{if ne .DriverName "mssql" -}}
```

Comment:
This is a pretty unhappy section. Before it was just cache/ret sections. But now it's much stranger.

We could actually have the cache.query = `insert into %%s values (%s)`, which leaves the cache.query as a format string set up for the ret portion. Then if there's a retMapping, we replace `%s` with `OUTPUT INSERTED` etc. If not we simply replace it with empty string. That's the only way I can think to keep the code a bit cleaner (though the method to clean the code up is sort of eh). What do you think of this?